data-access/orgaan.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  query,
+  sparqlEscapeString,
+  sparqlEscapeDateTime,
+  sparqlEscapeUri,
+} from 'mu';
+import moment from 'moment';
+
+import { PUBLICATION_STATUS } from '../util/constants';
+
+export async function getActivePersonen(bestuursorgaanId: string) {
+  const draftPublicatieStatus = sparqlEscapeUri(PUBLICATION_STATUS.DRAFT);
+  const safeStatussen = [
+    sparqlEscapeUri(
+      'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/21063a5b-912c-4241-841c-cc7fb3c73e75', // Effectief
+    ),
+    sparqlEscapeUri(
+      'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/c301248f-0199-45ca-b3e5-4c596731d5fe', // Verhinderd
+    ),
+  ];
+
+  const effectiveEndDateQuery = `
+   PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+   PREFIX org: <http://www.w3.org/ns/org#>
+   PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+   SELECT (MAX(?mandatarisEinde) AS ?effectiveEndDate)
+   WHERE {
+     ?boi mu:uuid ${sparqlEscapeString(bestuursorgaanId)} .
+     ?boi org:hasPost / ^org:holds ?mandataris  .
+     ?mandataris mandaat:einde ?mandatarisEinde.
+   }`;
+
+  const effectiveEndDateResult = await query(effectiveEndDateQuery);
+  const effectiveEndDate =
+    effectiveEndDateResult?.results?.bindings[0]?.effectiveEndDate?.value ||
```

Comment:
We still have this helper, not sure if you where a fan of it
```suggestion
  const effectiveEndDate =  findFirstSparqlResult(effectiveEndDateResult)?.effectiveEndDate?.value || new Date();
```